src/Microsoft.DotNet.Cli.Utils/AnsiConsole.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
you should cherry-pick that instead of copy&paste or:
git commit -m "message" --author "name last name <email>"
easiest thing you can do is this:
```
git remote add krwq https://github.com/krwq/cli.git
git fetch --all
git cherry-pick 6e465818c5e2cc093a6be34ec304352c3fab0a16
```